src/3_branch.js

```diff
@@ -299,14 +311,37 @@ Branch.prototype['init'] = wrap(callback_params.CALLBACK_ERR_DATA, function(done
 		// Keep android titanium from calling close
 		if (self.keepAlive) { setTimeout(function() { self.keepAlive = false; }, 2000); }
 		done(err, data && utils.whiteListSessionData(data));
+	},
+
+	attachVisibilityEvent = function() {
+		var hidden, changeEvent;
+		if (typeof document.hidden != undefined) {
+			hidden = "hidden",
+			changeEvent = "visibilitychange";
```

Comment:
Check this out: https://developer.mozilla.org/en-US/docs/Web/Guide/User_experience/Using_the_Page_Visibility_API

I guess because the internet lords didn't like camel or snake case.